playground/nextjs-app-router/next.config.mjs

```diff
@@ -1,4 +1,8 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
-
-export default nextConfig;
+const nextConfig = {
+  typescript: {
+    ignoreBuildErrors: true,
```

Comment:
@Zizzamia this is what we landed on to get this working with our current build/link steps, but Bunday pointed me at how they handle local imports in `base-org/web` as a potential alternative:

- https://github.com/base-org/web/blob/master/apps/web/next.config.js#L167
- https://github.com/base-org/web/blob/master/apps/web/tsconfig.json#L34
- https://github.com/base-org/web/blob/master/apps/web/tsconfig.json#L11